packages/graphql/package.json

```diff
@@ -4,19 +4,22 @@
   "private": true,
   "scripts": {
     "build": "webpack --config webpack.prod.js --display-error-details --display-modules",
+    "dev": "ts-node-dev -r tsconfig-paths/register --inspect --transpile-only --ignore-watch node_modules src/index.local.ts",
     "prepare:build": "scripts/prepareBuild.sh",
-    "sam": "yarn copy:env:to:build && concurrently \"webpack --config webpack.prod.js --watch\" \"sam local start-api --port 5000 --template aws/prod.yml\"",
     "start": "webpack --config webpack.dev.js --progress",
-    "start:infra": "docker-compose up",
+    "sam": "yarn copy:env:to:build && concurrently \"webpack --config webpack.prod.js --watch\" \"sam local start-api --port 5000 --template aws/prod.yml\"",
+    "start:infra": "docker-compose up -d",
     "test": "jest",
     "test:watch": "jest --watch",
-    "update:schema": "babel-node --extensions \".es6,.js,.es,.jsx,.mjs,.ts,.tsx\" ./scripts/updateSchema.ts"
+    "update:schema": "ts-node-dev ./scripts/updateSchema.ts"
   },
   "dependencies": {
     "@entria/graphql-mongoose-loader": "^4.3.0",
     "@koa/router": "^8.0.6",
+    "awilix": "^4.3.4",
```

Comment:
Why these packages were added?

Reply:
Awilix is an injection dependency package that helps decouple several layers of the application. It is a great tool for implements some SOLID concepts.